### ISSUE TITLE
Syntax highlight of yanked text.

### ISSUE THREAD

**Issue by o175** [Malabarba/aggressive-indent-mode#93]
My emacs doesn't highlight syntax of pasted pieces of js code in web major mode when aggressieve-indent-mode  is on.

**Comment by Malabarba** [comment]
Yes, I've noticed that too. Unfortunately, I have little idea of how to start debugging tha.
